src/views/design/ScriptDetail/EditLabels.tsx

```diff
@@ -52,39 +53,43 @@ function EditLabels({ labels, onChange, state }: IPublicProps & IObserveProps) {
             )}
             <ClickAwayListener onClickAway={handleClickAway}>
                 <div>
-                    <ButtonWithContent
-                        buttonText={<Translate msg="scripts.detail.side.labels.add_button" />}
-                        isOpen={isAddLabelFormOpen}
-                        onOpenIntent={() => setIsAddLabelFormOpen(true)}
-                        onCloseIntent={() => setIsAddLabelFormOpen(false)}
-                    >
-                        <TextInput
-                            id="new-label-name"
-                            label={translator('scripts.detail.side.labels.add_new.name')}
-                            required
-                            error={hasSubmitErrors && newLabelName === ''}
-                            value={newLabelName}
-                            onChange={(e) => setNewLabelName(e.target.value)}
-                        />
-                        <TextInput
-                            id="new-label-value"
-                            label={translator('scripts.detail.side.labels.add_new.value')}
-                            required
-                            error={hasSubmitErrors && newLabelValue === ''}
-                            value={newLabelValue}
-                            onChange={(e) => setNewLabelValue(e.target.value)}
-                        />
-                        <Box textAlign="right" marginTop={0.5}>
-                            <Button
-                                variant="contained"
-                                color="secondary"
-                                disableElevation
-                                onClick={handleSubmit}
+                    {checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC')
```

Comment:
Can we already use the security group name of the script in question instead of the fixed value 'PUBLIC'.